### ISSUE TITLE
[Enhancement] Data builder

### ISSUE THREAD

**Issue by MelanX** [Darkhax-Minecraft/BotanyPots#48]
It would be cool if you could add a recipe builder for the data for custom seeds. That would help to create the jsons for mods with a lot of seeds and custom farmland.

**Comment by Darkhax** [comment]
This sounds like a good idea, I have just never used data generators. Any examples of a mod with custom recipes I could look at?

**Comment by MelanX** [comment]
I think this could help: https://github.com/BluSunrize/ImmersiveEngineering/blob/1.15/src/main/java/blusunrize/immersiveengineering/api/crafting/ClocheRecipe.java

**Comment by Darkhax** [comment]
I've played around with data generators a bit and I don't think forge is stable enough for them yet. I wrote a generator for the vanilla crops but modded crops require the mods in the dev environment. CraftTweaker wants new MCP mappings while Botania wants old ones, on top of the many other errors I have had which keep delaying things. 

The serializer code is in place to do this, however I think I will wait until 1.16.2 or more likely 1.17 to revisit this.
